bob/src/grinder.rs

```diff
@@ -209,23 +210,39 @@ impl Grinder {
         self.hw_counter.spawn_task();
     }
 
-    pub(crate) async fn delete(&self, key: BobKey, with_aliens: bool) -> Result<u64, Error> {
+    pub(crate) async fn delete(&self, key: BobKey, options: DeleteOptions) -> Result<(), Error> {
         trace!(">>>- - - - - GRINDER DELETE START - - - - -");
-        counter!(CLIENT_DELETE_COUNTER, 1);
-        let sw = Stopwatch::start_new();
-        trace!(
-            "pass request to backend, /{:.3}ms/",
-            sw.elapsed().as_secs_f64() * 1000.0
-        );
-        let result = self.backend.delete(key, with_aliens).await;
-        trace!(
-            "backend processed delete, /{:.3}ms/",
-            sw.elapsed().as_secs_f64() * 1000.0
-        );
-        if result.is_err() {
-            counter!(CLIENT_DELETE_ERROR_COUNT_COUNTER, 1);
-        }
-        timing!(CLIENT_DELETE_TIMER, sw.elapsed().as_nanos() as f64);
+        let result = if options.force_node {
+            counter!(CLIENT_DELETE_COUNTER, 1);
+            let sw = Stopwatch::start_new();
+            trace!(
+                "pass request to backend, /{:.3}ms/",
```

Comment:
Add `delete` to text
```suggestion
                "pass delete request to backend, /{:.3}ms/",
```